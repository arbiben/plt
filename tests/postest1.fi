
int varia?
arr new_array?
char bestChar?

int func(){
varia = 1?
}


struct baby {
int age?
bool annoying?
}

int xi?
$ baby boy? $
