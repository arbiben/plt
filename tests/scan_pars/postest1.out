struct baby {
int age?
bool annoying?
}

int varia?
arr new_array?
char bestChar?
int a?
int xi?

int func()
{
varia = 1?
bestChar = 'y'?
bestChar = baby.age?
}

